runner/src/runner.rs

```diff
@@ -310,6 +339,19 @@ impl Runner {
     }
 }
 
+impl Drop for Runner {
```

Comment:
With the check that you have in `run`, is this still required? Having a long-running call (which the `_wait` implies it potentially is) inside a drop could potentially be quite confusing if you're not aware of it.

Reply:
I think many long-running functions implement `drop`, to clean up any open connections for instance. It's not required, but I like it as an added failsafe.